src/secrets/config.json

```diff
@@ -1,5 +1,6 @@
 {
   "admin_server": {
+    "api_key_init": "dba840fa0f7e03851ca695801f41c4a50fbbdb653ece0e2cdebbcb2977e5dfe1",
```

Comment:
Where did this key come from?

Reply:
I think that is the default key until you change the credentials.